api/v1alpha1/istio_types.go

```diff
@@ -68,6 +74,15 @@ func (s *IstioSpec) SetValues(values map[string]interface{}) error {
 	return nil
 }
 
+func (s *IstioSpec) GetRawValues() map[string]interface{} {
+	var rawVals map[string]interface{}
+	err := json.Unmarshal(s.RawValues, &rawVals)
+	if err != nil {
+		return nil
```

Comment:
We should print the error, or return it to the caller and handle it appropriately there. Same for `GetValues()` function above.